directives/tooltip/tooltip.test.js

```diff
@@ -0,0 +1,117 @@
+import { createLocalVue, mount } from '@vue/test-utils';
+import { DtTooltipDirective } from '@/directives';
+
+import {
+  TOOLTIP_DIRECTIONS,
+} from '@/components/tooltip/tooltip_constants';
+
+const MOCK_TRANSITION_STUB = () => ({
+  render: function (h) {
+    return this.$options._renderChildren;
+  },
+});
+const MOCK_TOOLTIP_TEXT = 'Tooltip text content';
+const MOCK_ANCHOR_TEXT = 'Button placeholder';
+
+const WrapperComponent = {
+  template: `
+    <button v-dt-tooltip:[placement]="MOCK_TOOLTIP_TEXT">{{MOCK_ANCHOR_TEXT}}</button>
+  `,
+  props: {
+    placement: {
+      type: String,
+      default: 'top',
+    },
+  },
+
+  data () {
+    return {
+      MOCK_ANCHOR_TEXT,
+      MOCK_TOOLTIP_TEXT,
+    };
+  },
+};
+
+const baseProps = {};
+
+let mockProps = {};
+
+const testContext = {};
+
+describe('DtTooltipDirective Tests', () => {
+  let wrapper;
+  let anchor;
+
+  const updateWrapper = () => {
+    wrapper = mount(WrapperComponent, {
+      localVue: testContext.localVue,
+      propsData: { ...baseProps, ...mockProps },
+      stubs: {
+        // this gets around transition async problems. See https://v1.test-utils.vuejs.org/guides/common-tips.html
+        transition: MOCK_TRANSITION_STUB(),
+      },
```

Comment:
should this be in a ...baseStubs, ...mockStubs type thing like the others?? I'm actually not sure.